docker/jenkins/Dockerfile.debian9-x86_64

```diff
@@ -0,0 +1,56 @@
+FROM debian:stretch
+
+RUN set -x \
```

Comment:
can you check with @stevenolen and see if there's an AWS region-specific cloudfront apt repo we could be hitting instead of the main Debian one? e.g. https://github.com/rstudio/rstudio/blob/master/docker/jenkins/Dockerfile.xenial-amd64#L6-L12